packages/openapi-generator/src/parser/document.spec.ts

```diff
@@ -16,7 +16,17 @@ describe('parseOpenApiDocument', () => {
     };
 
     const clonedInput = JSON.parse(JSON.stringify(input));
-    const parsedDocument = parseOpenApiDocument(input, 'TestService');
+    const parsedDocument = parseOpenApiDocument(
+      input,
+      'TestService',
+      'openapi/test-service.json',
+      {
+        'test-service': {
+          npmPackageName: '@sap/cloud-sdk-openapi-vdm-test-service',
+          directoryName: 'test-service'
+        }
+      }
+    );
```

Comment:
Could you make the assertion that the used package name and directory are the ones given in the mapping config.

Reply:
Also could you add a test that the service-mapping.json is written after one run. I think this is the case for the OData generator.

Reply:
As discussed, as initial version we only rely on the user input and will not write the file.